Recon/test/particleSources.py

```diff
@@ -0,0 +1,48 @@
+
+
+def get_cmds(part='e-', energyRange=(0.1,3.), thetaRangeDeg=(0.,20.), zpos=0.):
+    PI = 3.1415
```

Comment:
should this not come from some existing library's definition of pi? To avoid some comparison differences when somebody will do this with one more digit of pi and have slightly different result?

Reply:
now I do `from math import pi`